docs/source/030_installing.rst

```diff
@@ -150,14 +150,18 @@ to get Lmod installed quickly by using the defaults:
   the spider caching system.  This will greatly speed up ``module
   avail`` and ``module spider``
 
-To install Lmod, you'll want to carefully read the following.  If you
-want Lmod version X.Y installed in ``/opt/apps/lmod/X.Y``, just do::
+If you want Lmod version X.Y installed in ``/opt/apps/lmod/X.Y``, use
+the following form of ``configure``::
 
     $ ./configure --prefix=/opt/apps
```

Comment:
how about this:

```
$ export PREFIX=/opt/apps  # change this to your preference
$ ./configure --prefix=$PREFIX
```

and then using `$PREFIX` instead of `/opt/apps` below?